AppWidget/README.md

```diff
@@ -1,396 +1,81 @@
+In addition, the [MainActivity](app/src/main/java/com/example/android/appwidget/MainActivity.kt)
+showcases how to request the launcher to "pin" an appwidget.
 
-Implementing this feature allows for smoother scaling and overall better system health; this is because the system doesn't have to wake up the app every time it displays the widget in a different size.
+<img src="screenshots/todo-gif.gif" width="256px"
+alt="TODO widget showcase"
+title="Showcase of the TODO widget implementation" />
+<img src="screenshots/weather-gif.gif" width="256px"
+alt="Weather widget showcase"
+title="Showcase of the Weather widget implementation" />
+<img src="screenshots/images-gif.gif" width="256px"
+alt="Images widget showcase"
+title="Showcase of the Images widget implementation" />
 
-The following code examples show how to provide a list of layouts.
+> More showcasing resources in the [screenshots folder](screenshots)
 
-[`ListAppWidget.kt`](app/src/main/java/com/example/android/appwidget/ListAppWidget.kt\#L94)
+## Run Configuration
 
-```kotlin
-val viewMapping: MutableMap<SizeF, RemoteViews> = mutableMapOf()
-// Specify the maximum width and height in dp and a layout, which you want to use for the
-// specified size
-val viewMapping = mapOf(
-    SizeF(150f, 150f) to RemoteViews(
-        context.packageName,
-        R.layout.widget_grocery_list
-    ),
-    SizeF(250f, 150f) to RemoteViews(
-        context.packageName,
-        R.layout.widget_grocery_grid
-    ),
-)
-appWidgetManager.updateAppWidget(appWidgetId, RemoteViews(viewMapping))
-```
+When developing app widgets, create a new run configuration with:
 
-<img src="screenshots/widget_grocery_list.png"
-      alt="Screenshot of grocery list widget"
-      title="Screenshot of grocery list widget" />
+- Check "Always install with..."
+- Launch: Nothing
 
-<img src="screenshots/widget_grocery_grid.png"
-      alt="Screenshot of grocery grid widget"
-      title="Screenshot of grocery grid widget" />
+This will ensure that the widget is recreated but no activity is launched.
 
-# Pre-requisites
+<img src="screenshots/run-widget-config.png" width="512px"
+alt="screenshot showing run configuration in Android Studio for widgets"
+title="screenshot showing run configuration in Android Studio for widgets" />
 
-* Android SDK 31
+# Support
 
-# Getting Started
+- Stack Overflow: https://stackoverflow.com/questions/tagged/glance-appwidget
+- Channel #glance at KotlinLang
+  Slack [(request to join)](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up).
 
-This sample uses the Gradle build system. To build this project, use the
-"gradlew build" command or use "Import Project" in Android Studio.
+If you've found an error in this sample, please file an issue:
```

Comment:
Replace this line with:

If you've found an error in this sample, please file an issue at https://github.com/android/user-interface.

(and remove line 73)